extensions/kubernetes-service-binding/runtime/src/main/java/io/quarkus/kubernetes/service/binding/runtime/KubernetesServiceBindingConfigSourceProvider.java

```diff
@@ -30,6 +33,7 @@ public KubernetesServiceBindingConfigSourceProvider(String bindingRoot) {
         this.serviceBindingConverters = serviceBindingConverters;
         Path p = Paths.get(bindingRoot);
         if (!Files.exists(p)) {
+            log.warn("Binding root '" + p.toAbsolutePath().toString() + "' does not exist");
```

Comment:
I would say "Service Binding directory" , I found "Binding root" a bit out of context if you don't know the implementation details.